WSL/install-on-server.md

```diff
@@ -26,36 +24,48 @@ Before you can run Linux distros on Windows, you must enable the "Windows Subsys
 
 2. Restart your computer when prompted. **This reboot is required** in order to ensure that WSL can initiate a trusted execution environment.
 
+**If you're looking for 100% system call compatibility and faster IO performance, install WSL2!**
```

Comment:
We run into the same problem here as we did on the other page. I don't think we should show both install WSL2 and install WSL1 instructions on the same page. 

Here's a possible suggestion. WE can archive the old server install page (which is leaving it in the repo, but taking it off the table of contents) and add a link at the top 'IF you want to install just WSL1 then click here'. It's much much more unlikely that a user on a server SKU will have the major blockers to WSL2 (which is running VMWare or VirtualBox). Then this page will direct you to install WSL2 and will be much simpler.

Alternatively we can have a 'Install on Windows Server' and a 'Install just WSL1 on Window Server' but I like the earlier option more.